manifest/openshift4/coreos/ncp-openshift4.yaml

```diff
@@ -872,14 +871,6 @@ spec:
         - name: dir-tmp-usr-ovs-kmod-backup
           hostPath:
             path: /tmp/nsx_usr_ovs_kmod_backup
-
-
-        - name: host-usr-share
-          hostPath:
-            path: /usr/share
-        - name: host-depmod
-          hostPath:
-            path: /sbin/depmod
 {{else}}
```

Comment:
we can remove code line #L874 - 884.
{{else}}
..
#path: /tmp/nsx_usr_ovs_kmod_backup